vignettes/estimate_ascertainment.Rmd

```diff
@@ -0,0 +1,324 @@
+countries with large outbreaks to analyse. 
+
+## Downloading the raw data
+
+First of all, we subset the data so that we focus on just the first few months 
+of the COVID-19 outbreak in the U.K. We do so, as test availability changed 
+dramatically as a result of the vaccine campaign. We download the data — 
+using the `covidregionaldata` package — with the following 
+command:
+
+```{r, message = FALSE, warning = FALSE, eval = TRUE}
+df_covid_uk <- get_national_data(
+  countries = "united kingdom", source = "who", verbose = FALSE
+) |>
+  dplyr::rename(cases = cases_new, deaths = deaths_new)
+```
+
+We then subset the data to focus on just the first few months of the outbreak 
+with the following command:
+
+```{r, message = FALSE, warning = FALSE, eval = TRUE}
+df_covid_uk_subset <- subset(df_covid_uk, date <= "2020-05-31")
+```
+
+## Plotting the raw data
+
+First, we plot case incidence data with following command:
+
+```{r, message = FALSE, warning = FALSE, eval = TRUE, fig.cap = "\\label{fig:raw_data_covid} **Figure 1:** Incidence of cases over time for the ongoing COVID-19 outbreak in the U.K."}
+plot_case_data(df_covid_uk_subset)
+```
+
+Then, we plot the death incidence data with following command:
+
+```{r, message = FALSE, warning = FALSE, eval = TRUE, fig.cap = "\\label{fig:raw_data_covid} **Figure 2:** Incidence of  deaths over time for the ongoing COVID-19 outbreak in the U.K."}
+plot_death_data(df_covid_uk_subset)
+```
+
+## The delay distribution
+
+We again retrieve the appropriate distribution — reported 
+[here](https://doi.org/10.3390/jcm9020538)^[@linton2020incubation]^ — using the
+`epidist_db()` function from the `epiparameter` package, using the following
+command:
+
+```{r message=FALSE, warning=FALSE, eval=TRUE}
+onset_to_death_covid <- epidist_db(
+  disease = "COVID-19",
+  epi_dist = "onset_to_death",
+  author = "Linton_etal"
+)
+```
+
+To visualise this distribution, we evaluate it between 0 and 30 days, and plot
+the results over time. We do so using the following command:
+
+```{r message=FALSE, warning=FALSE, eval=TRUE, fig.cap = "\\label{fig:delay_dist_covid} **Figure 3: Example plot of the appropriate delay distribution for the COVID-19 epidemic in the U.K.** We plot the onset-to-death distribution we use throughout this example for COVID-19, reported in https://doi.org/10.3390/jcm9020538."}
+plot_epiparameter_distribution(onset_to_death_covid, by = 0.1)
+```
+
+## Estimating the proportion of cases that have been ascertained
+
+We use the `estimate_reporting()` function within the `datadelay` package to 
+calculate the time-varying CFR for the COVID-19 epidemic in the U.K:
+
+```{r message = FALSE, warning = FALSE, eval = TRUE}
+df_reporting_static <- estimate_reporting(df_covid_uk_subset,
+  epi_dist = onset_to_death_covid,
+  type = "static",
+  severity_baseline = 0.014,
+  correct_for_delays = TRUE
+) |>
+  format_output(estimate_type = "reporting", type = "Under-reporting")
```

Comment:
"Percent ascertained" might be clearer than "under-reporting", if type is easily changeable.

Reply:
Now converted to issue #26